.changeset/shy-snails-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+"@justeattakeaway/pie-modal": patch
+---
+
+[Fixed] - scroll lock behaviour for Safari
```

Comment:
Please could you mention what behaviour you've fixed/the solution a little just so we have a clearer understanding if we ever need to look back at the changelog?